src/test/java/com/wikia/webdriver/testcases/logintests/ForcedLoginTests.java

```diff
@@ -98,8 +98,8 @@ public void anonCanLoginViaUserLoginPage() {
     Assertion.assertTrue(special.isStringInURL(URLsContent.SPECIAL_UPLOAD));
   }
 
-  @Test(groups = "ForcedLogin_004_notLoggedIn")
-  public void anonCanLoginOnSpecialWatchListPage() {
+  @Test(groups = "Login_anonCanLogInOnSpecialWatchListPage")
+  public void anonCanLogInOnSpecialWatchListPage() {
```

Comment:
if we decide to use `LogIn` which seems to be right we should be consistent with all method names ;)